### PR TITLE
Add links to other instances

### LINKED ISSUE #982: Add links to other LPL instances
## What issue are you seeing?
Add links to other LPL instances on the main landing page.
Other instances:
- Seattle: https://openoversight.tech-bloc-sea.dev/
- Virginia: https://openoversightva.org/
- Baltimore: https://bpdwatch.com/
- Portland: https://cops.photo/ (no longer active)

### PR DESCRIPTION
**Pull request by michplunkett**
## Fixes issue
https://github.com/lucyparsons/OpenOversight/issues/982

## Description of Changes
Added links lo all known working instances of OpenOversight.

## Screenshots (if appropriate)
Screenshot 2024-10-08 at 3 22 38 PM

Screenshot 2024-10-08 at 3 22 46 PM

## Tests and Linting
- [x] This branch is up-to-date with the `develop` branch.
- [x] `pytest` passes on my local development environment.
- [x] `pre-commit` passes on my local development environment.

### CODE REVIEW

File: OpenOversight/app/templates/about.html [hunk 1]
```diff
@@ -40,10 +40,9 @@
         <div class="col-lg-4">
           <div class="col-half-border">
             <span class="fa fa-map-marker fa-2x"></span>
+            <p>It is the first project of its kind in the United States, and was first implemented in Chicago in October 2016.</p>
             <p>
-              It is the first project of its kind in the United States, and was first implemented in Chicago in October 2016.
-              OpenOversight launched in the East Bay of the San Francisco Bay Area in fall 2017 and in New York City in 2018. A
-              Baltimore instance was launched in 2019 at <a href="https://bpdwatch.com">BPDWatch.com</a>.
+              There are several instances up in other areas throughout the United States: <a href="https://bpdwatch.com/" target="_blank">Baltimore</a>, <a href="https://openoversight.tech-bloc-sea.dev/" target="_blank">Seattle</a>, and <a href="https://openoversightva.org/" target="_blank">Virginia</a>.
```
Comment (sea-kelp) [inline OpenOversight/app/templates/about.html]: Can we change this to:
```suggestion
              There are several instances up in other areas throughout the United States: <a href="https://bpdwatch.com/" target="_blank">Baltimore</a>, <a href="https://spd.watch/" target="_blank">Seattle</a>, and <a href="https://openoversightva.org/" target="_blank">Virginia</a>.
```

**Review by sea-kelp — APPROVED** [review]
One minor comment, otherwise looks good to me!

File: OpenOversight/app/templates/about.html (hunk 1 above)
Comment (michplunkett) [reply OpenOversight/app/templates/about.html]: Done done done!

**Comment by b-meson** [comment]
Thanks for sending this patch in. I agree its a better use of the language in there than the historical information that used to be there.

**Pull request merged**